docs/esa/jwst.rst

```diff
@@ -0,0 +1,751 @@
+.. doctest-skip-all
+
+.. _astroquery.esa.jwst:
+
+*********************************
+JWST TAP+ (`astroquery.esa.jwst`)
+*********************************
+
+The James Webb Space Telescope (JWST) is a collaborative project between NASA,
+ESA, and the Canadian Space Agency (CSA). Although radically different in
+design, and emphasizing the infrared part of the electromagnetic spectrum,
+JWST is widely seen as the successor to the Hubble Space Telescope (HST).
+The JWST observatory will consist of a deployable 6.6 meter passively cooled
```

Comment:
maybe rephrase for the present tense from the future, after all this module will only be used when JWST is already operational

Reply:
I have rephrased it, thanks for looking into this!